### ISSUE TITLE
Issues with testing in pr 589. Doc build fails with python 2.6, Sphnix issue

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#600]
Found the following in testing this morning for the 2.6 build:
```
0.74s$ make builddoc
./wbemcli --help >docs/wbemcli.help.txt
Done: Created wbemcli script help message file: docs/wbemcli.help.txt
./mof_compiler --help >docs/mof_compiler.help.txt
Done: Created mof_compiler script help message file: docs/mof_compiler.help.txt
rm -f build_doc/html/docs/index.html
PYTHONPATH=. sphinx-build -b html -v -d build_doc/doctrees -c docs -D latex_paper_size=a4 . build_doc/html
Error: Sphinx requires at least Python 2.7 or 3.4 to run.
make: *** [build_doc/html/docs/index.html] Error 1

The command "make builddoc" exited with 2.
```

See pr#589. for details of the failure.

I set it to priority because it blocks other testing

It happened in both travis tests but passed the apveyor tests

**Comment by andy-maier** [comment]
This issue was caused by Sphinx 1.5 refusing to work on Python 2.6.
The solution should be to skip any Sphinx building targets in the makefile if running on Python 2.6.

**Comment by andy-maier** [comment]
Fixed with PR #604 in master, and with PR #605 in stable 0.9.
